src/qt/rpcconsole.cpp

```diff
@@ -888,57 +888,69 @@ void RPCConsole::setMempoolSize(long numberOfTxs, size_t dynUsage)
 
 void RPCConsole::on_lineEdit_returnPressed()
 {
-    QString cmd = ui->lineEdit->text();
+    QString cmd = ui->lineEdit->text().trimmed();
 
-    if(!cmd.isEmpty())
-    {
-        std::string strFilteredCmd;
-        try {
-            std::string dummy;
-            if (!RPCParseCommandLine(nullptr, dummy, cmd.toStdString(), false, &strFilteredCmd)) {
-                // Failed to parse command, so we cannot even filter it for the history
-                throw std::runtime_error("Invalid command line");
-            }
-        } catch (const std::exception& e) {
-            QMessageBox::critical(this, "Error", QString("Error: ") + QString::fromStdString(e.what()));
-            return;
+    if (cmd.isEmpty()) {
+        return;
+    }
+
+    std::string strFilteredCmd;
+    try {
+        std::string dummy;
+        if (!RPCParseCommandLine(nullptr, dummy, cmd.toStdString(), false, &strFilteredCmd)) {
+            // Failed to parse command, so we cannot even filter it for the history
+            throw std::runtime_error("Invalid command line");
         }
+    } catch (const std::exception& e) {
+        QMessageBox::critical(this, "Error", QString("Error: ") + QString::fromStdString(e.what()));
+        return;
+    }
+
+    if (cmd.startsWith("stop")) {
```

Comment:
nit: `QString::startsWith()` has overload taking ` QLatin1String` which is recommended to use if exists. It will not allocate new `QString`, and comparing to latin1 instead of utf-16 is cheaper memory-bandwith-wise.

Also, maybe `startsWith` is not needed at all as the `cmd` string was trimmed beforehand? i.e. `cmd == QLatin1String("stop")` would suffice?

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/123#issuecomment-803235476).